ibis/backends/clickhouse/__init__.py

```diff
@@ -31,12 +38,77 @@
     _default_compression = False
 
 
-class Backend(BaseSQLBackend):
+class ClickhouseTable(ir.Table):
+    """References a physical table in Clickhouse."""
+
+    @property
+    def _qualified_name(self):
+        return self.op().name
+
+    @property
+    def _client(self):
+        return self.op().source
+
+    def invalidate_metadata(self):
+        self._client.invalidate_metadata(self._qualified_name)
+
+    def metadata(self) -> Any:
+        """Return the parsed results of a `DESCRIBE FORMATTED` statement.
+
+        Returns
+        -------
+        TableMetadata
+            Table metadata
+        """
+        return self._client.describe_formatted(self._qualified_name)
+
+    describe_formatted = metadata
+
+    @property
+    def name(self):
+        return self.op().name
+
+    def insert(self, obj, **kwargs):
```

Comment:
Shall we move this implementation to `_client` (the clickhouse backend) similarly to `desribte_formatted()` and `invalidate_metadata()`

Reply:
We can delete the `invalidate_metadata` impl here because it's not actually implemented.

Reply:
similarly with `describe_formatted`

Reply:
Removed the dead methods.

Let's keep `insert` here for now.